src/AddOrder.js

```diff
@@ -1,200 +1,373 @@
-import React from "react";
-import { useState } from "react";
-import { useCallback } from "react";
+import React, { useCallback } from "react";
 import Card from "react-bootstrap/Card";
 import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
+import Spinner from "react-bootstrap/Spinner";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
 import FloatingLabel from "react-bootstrap/FloatingLabel";
+import { InputGroup } from "react-bootstrap";
```

Comment:
```js
import InputGroup from "react-bootstrap/InputGroup";
```

Reply:
Dla utrzymania stylu. Też chyba jest to pewna optymalizacja bo nie importuje zbędnych rzeczy.

Reply:
Ale znów funkcje się importuje w klamrach. To chyba tylko dla stylu jest.